### PR TITLE
Add vCenter cluster automation scripts

### PR DESCRIPTION
**Pull request by dougm**
In support of issue #448

### CODE REVIEW

File: machines/vcsa/create-cluster.sh [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Configure a vCenter cluster with vSAN datastore, DVS and DVPGs
+
+export GOVC_INSECURE=1
+export GOVC_USERNAME=${GOVC_USERNAME:-"Administrator@vsphere.local"}
+if [ -z "$GOVC_PASSWORD" ] ; then
+    # extract password from $GOVC_URL
```
Comment (dougm) [inline machines/vcsa/create-cluster.sh]: Too much parsing of GOVC_URL here and other scripts.  Made this PR to simplify: https://github.com/vmware/govmomi/pull/497

Just need to figure out why Travis and Drone are unhappy.

File: machines/vcsa/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+go get -u github.com/vmware/govmomi/govc
+```
+
+### jq
+
+Used here to derive static VCSA networking from its parent ESXi host.
+But, you should already be using and loving jq for other tasks: http://stedolan.github.io/jq/
+
+```
+brew install jq
```
Comment (fdawg4l) [inline machines/vcsa/README.md]: `$ sudo apt-get install jq` on Ubuntu

File: machines/vcsa/README.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+Note that for a cluster to use vSAN, it will need at least 3 of these
+ESXi VMs.
+
+### create-vcsa-vm.sh
+
+This script creates a VM with VCSA (Virtual Center Server Appliance)
+installed.
+
+### create-cluster.sh
```
Comment (fdawg4l) [inline machines/vcsa/README.md]: What does cluster here mean specifically?  Does it mean create the VSAN datastore across the esxi instances and attach the esxi instances to the VC created above?  Or something else?

File: machines/vcsa/create-cluster.sh (hunk 1 above)
Comment (caglar10ur) [inline machines/vcsa/create-cluster.sh]: govet is no longer under golang.org/x/tools/cmd/vet. Not sure why

Comment (caglar10ur) [inline machines/vcsa/create-cluster.sh]: https://github.com/vmware/vic/pull/534

**Comment by caglar10ur** [comment]
LGTM :sheep: :it:

File: machines/vcsa/create-cluster.sh (hunk 1 above)
Comment (dougm) [inline machines/vcsa/create-cluster.sh]: fixed in vmware/govmomi#498

File: machines/vcsa/README.md (hunk 3 above)
Comment (dougm) [inline machines/vcsa/README.md]: I had left the doc within the script, but added to the README section

File: machines/vcsa/README.md (hunk 2 above)
Comment (dougm) [inline machines/vcsa/README.md]: removed install instructions, just have urls now

**Comment by dougm** [comment]
@fdawg4l @caglar10ur pushed some changes after your reviews, ok to merge?

**Comment by fdawg4l** [comment]
LGTM

**Comment by caglar10ur** [comment]
Yep :sheep: :it:

**Pull request merged**